minecode/mappers/debian.py

```diff
@@ -21,8 +21,8 @@
 from minecode import map_router
 from minecode.mappers import Mapper
 from minecode.utils import form_vcs_url
-# from minecode import debutils
-
+from minecode import debutils
+from packagedb.models import DependentPackage
```

Comment:
I don't think we need this import. We use `DependentPackage` from scancode, rather than the packagedb.